packages/react-components/react-spinbutton/src/stories/SpinButtonAppearance.stories.tsx

```diff
@@ -2,36 +2,36 @@ import * as React from 'react';
 import { SpinButton } from '../index';
 import { Label } from '@fluentui/react-label';
 import { useId } from '@fluentui/react-utilities';
-import { Text } from '@fluentui/react-text';
-import { makeStyles, shorthands } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import { tokens } from '@fluentui/react-theme';
 
 const useStyles = makeStyles({
   base: {
     display: 'flex',
     flexDirection: 'column',
     maxWidth: '500px',
+  },
 
-    '> div': {
-      display: 'flex',
-      flexDirection: 'column',
-      marginTop: tokens.spacingVerticalMNudge,
-    },
+  field: {
+    display: 'grid',
+    gridRowGap: tokens.spacingVerticalXXS,
+    marginTop: tokens.spacingVerticalMNudge,
+    ...shorthands.borderRadius(tokens.borderRadiusMedium),
```

Comment:
Out of curiosity, why does this have a border radius? Is there some design or recommended visual appearance that uses a border radius like this? If not, it might be better to leave it out so people don't think it's a recommended design.

Reply:
The best practices in Figma show the controls against a background with rounded corners but I agree with you, doing this implies it's a recommendation and I don't think that it is.

I've taken it out for now, we can always add it back later if necessary.